rpa-bot/1.3/modules/ROOT/pages/_partials/hard-soft-requirements.adoc

```diff
@@ -28,12 +30,28 @@ include::rpa-home::partial$windows-versions-support.adoc[]
 
 |===
 
-=== Requirements for Secure Session Technology
+=== System Settings
+
+* Display resolution: 
+** Minimum: 1280 x 1080
+** Recommended: Full HD 1080p (1920 × 1080) or higher
+* Scaling of text and other items on the screen: 100 %
+* _Clear Type_: turned off
```

Comment:
```suggestion
* _Clear Type_: disabled
```

Reply:
The Windows UI uses 'turn on' -> I'm leaving that (BTW I don't think it's confusing in general for the reader to read phrasal verbs)